examples/social/tests/cross_chain.rs

```diff
@@ -48,14 +48,24 @@ async fn test_cross_chain_posting() {
     chain1.handle_received_effects().await;
 
     // Querying the own posts
-    let query_string = "query { ownPosts(start: 0, end:1) { timestamp, text } }";
-    let value: serde_json::Value = chain2.query(application_id, query_string).await;
-    let value = value["data"]["ownPosts"][0]["text"].clone();
+    let query = "{ ownPosts(start: 0, end: 1) { timestamp, text } }";
+    let response = chain2
+        .query(application_id, query.into())
+        .await
+        .data
+        .into_json()
+        .expect("Unexpected non-JSON query response");
+    let value = response["ownPosts"][0]["text"].clone();
     assert_eq!(value, "Linera is the new Mastodon".to_string());
 
     // Now handling the received messages
-    let query_string = "query { receivedPostsKeys { timestamp, author, index } }";
-    let value: serde_json::Value = chain1.query(application_id, query_string).await;
-    let author = value["data"]["receivedPostsKeys"][0]["author"].clone();
+    let query = "{ receivedPostsKeys { timestamp, author, index } }";
```

Comment:
Is the outer pair of curly braces required?

Reply:
This might be a shorthand for `query { .. }`.